Docker/build/WikibaseBundle/LocalSettings.WikibaseLocalMedia.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+// https://github.com/ProfessionalWiki/WikibaseLocalMedia
+
+## WikibaseLocalMedia Configuration
+wfLoadExtension( 'WikibaseLocalMedia' );
```

Comment:
Wont this cause problems on client installations, unless someone else does the same we did below?

Probably should be guarded by `ExtensionRegistry::getInstance()->isLoaded( 'WikibaseRepository' )`.

Reply:
I tried this but it did not get this to work consistently on both REL1_35 and master but lets try again and fix it otherwise.

Reply:
Ok sadly this does not work in the LocalSettings.php file. It will not be loaded at that point in time but rather be in some queue to be loaded.

I dropped the if case and excluded it in the loading of the repo-client testing instead.